.travis.yml

```diff
@@ -8,6 +8,13 @@ notifications:
       - "irc.freenode.org#2600hz-dev"
 
 otp_release:
+  - 17.5
+  - 17.4
+  - 17.3
+  - 17.1
+  - 17.0
+  - R16B02
+  - R16B01
```

Comment:
@lazedo It's for testing actually. Look at this build for example: https://travis-ci.org/2600hz/kazoo/builds/77743535

And it doesn't cost anything to test all versions…